agent/engine/docker_task_engine.go

```diff
@@ -975,12 +975,10 @@ func getLogRouterConfig(task *apitask.Task, container *apicontainer.Container, h
 	fields := strings.Split(task.Arn, "/")
 	taskID := fields[len(fields)-1]
 	tag := container.Name + "-" + taskID
-	fluentd := filepath.Join(socketPathPrefix, cfg.DataDirOnHost, dataLogDriverPath, taskID, dataLogDriverSocketPath)
+	fluentd := socketPathPrefix + filepath.Join(cfg.DataDirOnHost, dataLogDriverPath, taskID, dataLogDriverSocketPath)
```

Comment:
what's the difference between before/after change?

Reply:
We want to keep the Unix:/// , the filepath join removes the extra //